README.md

```diff
@@ -11,13 +11,33 @@ Help us build a Credit Card Approval System - Using Machine Learning!
 
 <p>
   <a href="https://acmvit.in/" target="_blank">
-    <img alt="made-by-acm" src="https://img.shields.io/badge/MADE%20BY-ACM%20VIT-blue?style=for-the-badge" />
   </a>
-    <!-- Uncomment the below line to add the license badge. Make sure the right license badge is reflected. -->
-    <!-- <img alt="license" src="https://img.shields.io/badge/License-MIT-green.svg?style=for-the-badge" /> -->
-    <!-- forks/stars/tech stack in the form of badges from https://shields.io/ -->
+  <img alt="made-by-acm" src="https://img.shields.io/badge/MADE%20BY-ACM%20VIT-blue?style=for-the-badge"/>
+  <img alt="license" src="https://img.shields.io/badge/License-MIT-green.svg?style=for-the-badge"  />
+  <img alt="stars" src="https://img.shields.io/github/stars/ACM-VIT/Good-Client-Bad-Client?style=social" align="right"/> 
+  <img alt="forks" src="https://img.shields.io/github/forks/ACM-VIT/Good-Client-Bad-Client?style=social" align="right"/>   
+  
+    
 </p>
 
+## Overview
+
+The main motive of the project is to build a machine learning model to predict if an applicant is 'good' or 'bad' client, different from other tasks, the definition of 'good' or 'bad' is not given. <br>
+Credit score cards are a common risk control method in the financial industry. It uses personal information and data submitted by credit card applicants to predict the probability of future defaults and credit card borrowings. The bank is able to decide whether to issue a credit card to the applicant. Credit scores can objectively quantify the magnitude of risk.<br>
+    In dataset,application_record.csv is the table/file that has information about all the customers regarding their socio-economic status and credit_record.csv is the file/table that has all the payment/default records for a given client.<br>
+
+
+---
+
+## Usage
+Run the following command on your terminal to install are the packages related to this project
```

Comment:
- change this to Run the following command to install all the required packages for this project
- do not comment out the initial usage steps (git clone/fetch etc) its important